scripts/brushless.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Can you explain / add documentation explaining what this is? also, is it even necessary

Reply:
Like if anything, maybe just have a setup_pi script

Reply:
Yea, after second thought, I think just add documentation and then just add it to the bashrc. I think the other software leads can comment on this

Reply:
is all of this necessary because of sudo privileges? I assume you can't just put these things in ~/.bashrc because the environment variables from there don't carry over once you switch to root. if that's the case, we should see if its possible to setup a bashrc file for the root user, and then put all these environment variables into a setup file which we source from there

Reply:
Yea there was issues with the environment variables carrying over. The bashrc file runs on startup right? If there exists one with root privileges then we can just launch the whole ros node on startup of the pi (unless bashrc is not a good place to put the launching of ros nodes). Otherwise yea if we can source those environment variables in root then maybe roslaunch with sudo will work. I'm reading something about logging in as sudo `sudo su` and then just looking in `~/.bashrc` but I'm not sure. I can mess with this at some point unless we have something specific to try first.

Reply:
as an update, it seems like alan resolved this and now we can just ssh into root@10.0.0.3 and then just run roslaunch mrover brushless/pi.launch normally

Reply:
yup I think we should get rid of this file unless you guys still want it for something

Reply:
big fan of deleting it, alan said that he'll add to the wiki soon on how to set stuff up so we should be good deleting this

Reply:
Sounds good. I have made a wiki under Dev-Ops called "Setting up the Pi".

Reply:
Thanks for updating the wiki Alan!